README.md

```diff
@@ -1,10 +1,10 @@
 <a name="documentr_top"></a>[![official JetBrains project](https://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub) [![Build Status](https://api.cirrus-ci.com/github/JetBrains/gradle-intellij-plugin.svg)](https://cirrus-ci.com/github/JetBrains/gradle-intellij-plugin)
  [![Twitter Follow](https://img.shields.io/twitter/follow/JBPlatform?style=flat)](https://twitter.com/JBPlatform/)
- [![Gradle Plugin Release](https://img.shields.io/badge/gradle%20plugin-0.4.21-blue.svg)](https://plugins.gradle.org/plugin/org.jetbrains.intellij) [![GitHub Release](https://img.shields.io/github/release/jetbrains/gradle-intellij-plugin.svg)](https://github.com/jetbrains/gradle-intellij-plugin/releases) 
+ [![Gradle Plugin Release](https://img.shields.io/badge/gradle%20plugin-0.4.22-blue.svg)](https://plugins.gradle.org/plugin/org.jetbrains.intellij) [![GitHub Release](https://img.shields.io/github/release/jetbrains/gradle-intellij-plugin.svg)](https://github.com/jetbrains/gradle-intellij-plugin/releases) 
```

Comment:
readme is generated by `:documetr` task, versions will be fixed automatically, sources are in `docs/` directory

Reply:
ok, is this the only place, which is automatically updated, or are there are places which I've unnecessarily updated?

Reply:
all places will be updated

Reply:
You have to repeat this for almost every new project contributor. Could we add a comment mentioning `:documetr` into each generated document, or something like that? AFAIK, Markdown should properly support HTML comments.

Reply:
All of the version changes are removed.

Reply:
@piotrtomiak `docs` directory is still untouched, the entire README will be overwritten and changes you've made will be lost. Please make changes in `docs` dir and run `:documentr` task.

@ForNeVeR only README.md is generated. Let's try

Reply:
@zolotov sorry, I missed the point. Should be fine now.